xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -37,19 +42,15 @@ constexpr const char* userFontPath = "special://home/media/Fonts/";
 // Directory where Kodi bundled fonts (default ones like Arial or Teletext) are located
 constexpr const char* systemFontPath = "special://xbmc/media/Fonts/";
 
-std::string GetDefaultFontPath(std::string& font)
+std::string GetSystemFontPath(std::string font)
 {
-  constexpr std::array<const char*, 2> fontSources{userFontPath, systemFontPath};
-
-  for (const auto& path : fontSources)
+  std::string fontPath = URIUtils::AddFileToFolder(systemFontPath, font);
+  if (XFILE::CFile::Exists(fontPath))
   {
-    auto fontPath = URIUtils::AddFileToFolder(path, font);
-    if (XFILE::CFile::Exists(fontPath))
-    {
-      return CSpecialProtocol::TranslatePath(fontPath).c_str();
-    }
+    return CSpecialProtocol::TranslatePath(fontPath).c_str();
```

Comment:
You're returning cstr here but the signature of the method returns STD::string. Is the conversion intended?

Reply:
forgot thanks